### ISSUE TITLE
Create a more cohesive color theme for node types

### ISSUE THREAD

**Issue by keyserj** [amelioro/ameliorate#526]
### Describe your issue

Many of the colors for node types (all can be seen at https://ameliorate.app/examples/ontology) are too dark to easily read the node type's text (or the node's text, if using the `fill nodes with color` options). They also vary in levels of saturation.

### Solution you'd like

Using the [new oklch color notation](https://evilmartians.com/chronicles/oklch-in-css-why-quit-rgb-hsl), it should be much easier to pick new colors for each node type such that:
- they have consistent Lightness and Chroma (saturation)
- the Lightness has sufficient contrast to easily read black text
- Chroma can be increased to highlight the main two node types (problem & solution)
- for #68, Lightness/Chroma can be adjusted to create dark-mode colors for the same hues

### Additional context

- [This blog post](https://lea.verou.me/blog/2024/contrast-color/#does-this-mythical-l-threshold-actually-exist%3F) has a nice demo where you can adjust oklch and see if the color passes APCA/WCAG 2.1 on white/black text. It seems like 75 Lightness and 0.12 Chroma enable all Hue values to pass on black text, and 0.6 Lightness and 0.10 Chrome enable all Hue values to pass on white text (at least for AA).
- [EvilMartian's oklch color picker](https://oklch.com/#60,0.1005,0,100) makes it easy to see which values are possible to be rendered on today's monitors.

### Technical ideas and questions

- bottom of [oklch article](https://evilmartians.com/chronicles/oklch-in-css-why-quit-rgb-hsl) mentions [stylelint](https://stylelint.io/) which can use linting to ensure we're picking colors that can be rendered by monitors

**Comment by keyserj** [comment]
Looks like stylelint is intended for actual CSS files, and I had troubles setting it up for a ts file. Could spend more effort but it's not a big deal.
